### PR TITLE
Debug middleware

### PR DESCRIPTION
**Pull request by yanns**
Same as https://github.com/tokio-rs/axum/pull/1993, rebased on `main`

### CODE REVIEW

**Comment by yanns** [comment]
Issue with this PR:
```
test axum-macros/src/lib.rs - debug_middleware (line 634) has been running for over 60 seconds
```

File: axum/src/lib.rs [hunk 1]
```diff
@@ -463,7 +463,7 @@ pub use self::form::Form;
 pub use axum_core::{BoxError, Error, RequestExt, RequestPartsExt};
 
 #[cfg(feature = "macros")]
-pub use axum_macros::debug_handler;
+pub use axum_macros::{debug_handler, debug_middleware};
```
Comment (mladedav) [inline axum/src/lib.rs:466]: This reexport should probably also go to the `axum` changelog.

Comment (yanns) [reply axum/src/lib.rs:466]: https://github.com/tokio-rs/axum/pull/2725/commits/db0b09ec3622ad61f7cd20d4829f18fdb231e8e4

**Review by jplatte — APPROVED** [review]
Thanks for your help finishing this!

**Pull request merged**